go/extra/ba/README.md

```diff
@@ -0,0 +1,10 @@
+# Benchmark analysis
+
+Connects to prometheus and compares results of benchmark results.
```

Comment:
"compares results of benchmark results"?

Reply:
I rewrote the readme with actual basic usage cases.